lib/donos/bot/logic.ex

```diff
@@ -60,6 +60,16 @@ defmodule Donos.Bot.Logic do
     Session.start(message.from.id)
   end
 
+  def handle_post({:command, <<"setname", name :: binary>>}, message, _reply_to) do
```

Comment:
add space after setname
`/setnamekana` is valid command now, but it shouldn't be